README.md

```diff
@@ -1,15 +1,15 @@
-# README
+# Hangman Web Application Game
 
-This is a simple web application game of Hangman
+This is a simple web application game of Hangman.
 
 In order to run this game on your local system, keep these specs and steps in mind:
 
-* Ruby versions 2.7.0 or later to run Rails
+* Ruby versions 2.7.0 or later is required to support Rails.
 
-* Use ```gem install rails``` to install rails 7.0.0 or later
+* Use ```gem install rails``` to install rails 7.0.0 or later, if not previously installed.
 
-* Use ```bundle install``` to run the Gemfile and install/update all the required gems from the app directory ```cd HangmanRailsGame```
+* From the HangmanRailsGame directory use ```bundle install``` to run the Gemfile and install/use all the required gems.
 
-* To start the web server to run your web application use ```rails server``` command.
+* To start the web server to run your web application use ```rails server```.
```

Comment:
`rails server` is correct but we also can do `rails s` instead :)

Reply:
Thanks for the inputs, will make the necessary changes.